docs/source/examples.rst

```diff
@@ -407,6 +407,72 @@ Plotting with Manim
 
             self.add(ax, labels, curve_1, curve_2, line_1, line_2, riemann_area, area)
 
+.. manim:: PolygonOnAxes
+
+    class PolygonOnAxes(Scene):
+        def get_rectangle_coors(self, bottom_left, top_right):
+            return [
+                (top_right[0], top_right[1]),
+                (bottom_left[0], top_right[1]),
+                (bottom_left[0], bottom_left[0]),
+                (top_right[0], bottom_left[0]),
+            ]
+
+        def construct(self):
+            ax = Axes(
+                x_range=[0, 10],
+                y_range=[0, 10],
+                x_length=6,
+                y_length=6,
+                axis_config={"include_tip": False},
+            )
+
+            t = ValueTracker(5)
+            k = 25
+
+            graph = always_redraw(
+                lambda: ax.plot(
+                    lambda x: k / x,
+                    color=YELLOW_D,
+                    x_range=[
+                        k / 10,
+                        10.0,
+                        0.01,
+                    ],
+                    use_smoothing=False,
+                )
+            )
```

Comment:
Given that none of the parameters used for the graph are animated, this can be simplified a bit:

```suggestion
            graph = ax.plot(
                lambda x: k / x,
                color=YELLOW_D,
                x_range=[k / 10, 10.0, 0.01],
                use_smoothing=False,
            )
```